src/screens/tutorial/views/ItemView.tsx

```diff
@@ -1,21 +1,32 @@
-import React from 'react';
-import {StyleSheet, Image, ImageSourcePropType} from 'react-native';
+import React, {useRef, useLayoutEffect} from 'react';
+import {StyleSheet, Image, ImageSourcePropType, AccessibilityInfo, findNodeHandle} from 'react-native';
 import {Text} from 'components';
 import {useI18n} from '@shopify/react-i18n';
 
 import {TutorialKey} from '../TutorialContent';
 
-interface ItemViewProps {
+export interface ItemViewProps {
   item: TutorialKey;
   image: ImageSourcePropType;
+  isActive: boolean;
 }
 
-export const ItemView = ({item, image}: ItemViewProps) => {
+export const ItemView = ({item, image, isActive}: ItemViewProps) => {
   const [i18n] = useI18n();
   const bodyText = i18n.translate(`Tutorial.${item}`).split(/\n\n/g);
+  const imageRef = useRef<any>();
+
+  useLayoutEffect(() => {
+    const tag = findNodeHandle(imageRef.current);
+    if (isActive && tag) {
+      AccessibilityInfo.setAccessibilityFocus(tag);
```

Comment:
So that's how you do that! Awesome. We will be using this other places in the app as well.

Reply:
This is for auto focus. The but main fix for Carousal is `importantForAccessibility="no"` for Android and `accessibilityElementsHidden={index !== currentStep}` for iOS